src/lib/y2storage/dialogs/inst_disk_proposal.rb

```diff
@@ -36,6 +36,20 @@ class InstDiskProposal < ::UI::InstallationDialog
       def initialize
         super
         textdomain "storage"
+        propose
```

Comment:
np: I would rename it to `propose!` as the only effect of this method is to change the object's internal state.